src/esp/gfx/Simulator.cpp

```diff
@@ -158,7 +161,10 @@ void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
         if (endsWith(houseFilename, ".house")) {
           scene::SemanticScene::loadMp3dHouse(houseFilename, *semanticScene_);
         } else if (endsWith(houseFilename, ".scn")) {
-          scene::SemanticScene::loadGibsonHouse(houseFilename, *semanticScene_);
+          scene::SemanticScene::loadGibsonHouse(
+              houseFilename, *semanticScene_,
+              // Default World rotation for Gibson semantic data
+              quatf::FromTwoVectors(geo::ESP_GRAVITY, -vec3f::UnitZ()));
```

Comment:
mp3d does this as a default argument in the header file. Not sure which is better but we should try to be consistent

Reply:
@msbaines, that's great suggestion, thank you!